composer.json

```diff
@@ -35,6 +35,7 @@
         "symfony/dependency-injection": "^4.3",
         "symfony/doctrine-bridge": "^4.3",
         "symfony/event-dispatcher": "^4.3",
+        "symfony/event-dispatcher-contracts": "^1.1",
```

Comment:
Sorry, I said one major version, but it is compatible with version 2, we should allow it. :sweat_smile:  So… is it?

Reply:
See https://github.com/symfony/event-dispatcher/blob/4.3/composer.json#L20

Reply:
We need to adjust it for Symfony 5 compat then at some point:

https://github.com/symfony/event-dispatcher/blob/5.0/composer.json#L20

Reply:
Ok, it can be done when working on allowing sf 5.